tests/unit/dom/select-test.js

```diff
@@ -0,0 +1,210 @@
+import { module, test } from 'qunit';
+import { select, setupContext, teardownContext } from '@ember/test-helpers';
+import { buildInstrumentedElement } from '../../helpers/events';
+import { isIE11 } from '../../helpers/browser-detect';
+
+let selectSteps = ['focus', 'focusin', 'input', 'change'];
+let additionalSteps = ['input', 'change'];
+if (isIE11) {
+  selectSteps = ['focusin', 'input', 'change', 'focus'];
```

Comment:
These are a bit off, under IE11 we should have the following (the only difference is the ordering between `focus` and `focusin`):

```suggestion
  selectSteps = ['focusin', 'focus', 'input', 'change'];
```